src/main/java/io/github/seggan/slimypowersuits/Setup.java

```diff
@@ -19,13 +19,18 @@
 import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
 import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
 import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
 import org.bukkit.inventory.ItemStack;
 
 public class Setup {
 
     static void registerItems(SlimyPowerSuits plugin) {
+
+        new ElementForge(MiscItems.SUITS, MiscItems.ELEMENT_FORGE).register(plugin);
+
+        RecipeType ELEMENT_FORGE_RECIPE = new RecipeType(new NamespacedKey(SlimyPowerSuits.getInstance(), "element_forge_recipe"), MiscItems.ELEMENT_FORGE, "&cElement Forge", "&7Used to create powerful items");
```

Comment:
Eh, I don't know. Maybe something more useful like "Used to create new elements" or something?